Description: lib/: Fix paths
 Use '/var/log/schleuder' for logging.
 
 Set the 'root_dir' to '/usr/lib/ruby/vendor_ruby/schleuder/'.

 Additionally, add '/usr/share/schleuder/locales/*.yml' to the locales
 load path. 

 Use 'require' instead of 'require_relative'.

 Upstream is aware of this patch and we're working on integrating it
 upstream.
Author: Georg Faerber <georg@riseup.net>
Last-Update: 2018-02-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/schleuder/conf.rb
+++ b/lib/schleuder/conf.rb
@@ -9,7 +9,7 @@
 
     DEFAULTS = {
       'lists_dir' => '/var/lib/schleuder/lists',
-      'listlogs_dir' => '/var/lib/schleuder/lists',
+      'listlogs_dir' => '/var/log/schleuder/lists',
       'plugins_dir' => '/etc/schleuder/plugins',
       'log_level' => 'warn',
       'superadmin' => 'root@localhost',
--- a/lib/schleuder/cli.rb
+++ b/lib/schleuder/cli.rb
@@ -77,7 +77,7 @@
     desc 'install', "Set-up or update Schleuder environment (create folders, copy files, fill the database)."
     def install
       config_dir = Pathname.new(ENV['SCHLEUDER_CONFIG']).dirname
-      root_dir = Pathname.new(ENV['SCHLEUDER_ROOT'])
+      root_dir = Pathname.new('/usr/lib/ruby/vendor_ruby/schleuder/')
 
       # Check if lists_dir contains v2-data.
       if Dir.glob("#{Conf.lists_dir}/*/*/members.conf").size > 0
--- a/lib/schleuder.rb
+++ b/lib/schleuder.rb
@@ -75,6 +75,7 @@
 end
 
 I18n.load_path += Dir["#{rootdir}/locales/*.yml"]
+I18n.load_path += Dir["/usr/share/schleuder/locales/*.yml"]
 I18n.enforce_available_locales = true
 I18n.default_locale = :en
 
--- a/lib/schleuder-api-daemon.rb
+++ b/lib/schleuder-api-daemon.rb
@@ -7,7 +7,7 @@
 require 'sinatra/json'
 require 'sinatra/namespace'
 require 'thin'
-require_relative '../lib/schleuder.rb'
+require 'schleuder'
 
 
 %w[tls_cert_file tls_key_file].each do |config_key|
